x-pack/legacy/plugins/siem/public/components/timeline/body/events/stateful_event.tsx

```diff
@@ -166,10 +165,7 @@ export class StatefulEvent extends React.Component<Props, State> {
     // see componentDidMount() for when it schedules the first
     // time this stateful component should be rendered.
     if (!this.state.initialRender) {
-      // height is being inlined directly in here because of performance with StyledComponents
-      // involving quick and constant changes to the DOM.
-      // https://github.com/styled-components/styled-components/issues/134#issuecomment-312415291
-      return <EmptyRow style={{ height: DEFAULT_ROW_HEIGHT }}></EmptyRow>;
+      return <SkeletonRow cellCount={9} />;
```

Comment:
is `cellCount={9}` always valid?

Reply:
My intent with the `SkeletonRow` component was to give the impression to the user that tabular data would be loaded in. I was thinking the component might have uses elsewhere in the future, so I gave it a default of 4 cells, with the option to manually set the cell count using the `cellCount` prop. I settled on using `9` here because the number of default number of timeline data columns is 8, plus 1 for the actions column.

That said, it looks like I can easily change it to be dynamic count of the number of columns the user currently has in the timeline. I'll make that change now.

In the future, we can investigate enhancements such as making the `SkeletonRow` cells match the actual column widths, but I think this works for now. Thoughts?

Reply:
matching the (dynamic) count of the current number of columns makes sense 👍